code/templates/admin/admin-dashboard.html

```diff
@@ -26,6 +26,7 @@
                     <th id="password">Password</th>
                     <th id="email">Email</th>
                     <th id="phone">Phone</th>
+                    <th id="alerts_email">Zone</th>
```

Comment:
Why does this share an id with the line below it?

Reply:
Accident! The zone field didn't exist so I did a hasty Ctrl C, Ctrl V